scripts/node_integration_tests/playbooks/golem/task_timeout/playbook.py

```diff
@@ -29,12 +29,15 @@ class Playbook(NodeTestPlaybook):
     def step_wait_subtask_completed(self):
         def on_success(result):
             if result:
-                statuses = map(lambda s: s.get('status'), result)
-                if any(map(lambda s: s == 'Finished', statuses)):
+                statuses = list(map(lambda s: s.get('status'), result))
```

Comment:
```suggestion
                statuses = [s.get('status') for s in result]
```
This is what list comprehensions are for. Much clearer.